metadata/src/main/resources/common/metadata/UnfenceBrokerRecord.json

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 8,
+  "type": "metadata",
+  "name": "UnfenceBrokerRecord",
+  "validVersions": "0",
+  "fields": [
+    { "name": "Id", "type": "int32", "versions": "0+",
+      "about": "The broker ID to unfence." },
+    { "name": "Epoch", "type": "int64", "versions": "0+",
+      "about": "The epoch of the broker to unfence." }
```

Comment:
In `UnregisterBrokerRecord` and `RegisterBrokerRecord` we call this `BrokerId` and `BrokerEpoch` receptively. Should we use the same names here for consistency?

Reply:
ok